go-controller/pkg/ovn/egressqos_test.go

```diff
@@ -569,6 +692,8 @@ var _ = ginkgo.Describe("OVN EgressQoS Operations", func() {
 			i, n, _ := net.ParseCIDR("10.128.1.3" + "/23")
 			n.IP = i
 			fakeOVN.controller.logicalPortCache.add(podT, "", types.DefaultNetworkName, "", nil, []*net.IPNet{n})
+			// add pod to local zone (isPodScheduledinLocalZone logic depends on cache entry)
+			fakeOVN.controller.localZoneNodes.Store(podT.Spec.NodeName, true)
 
```

Comment:
can you add a non-local pod to verify it doesn't get handled?

Reply:
good point... let me do that.

Reply:
this is done.